packages/syntax-core/src/SelectList/SelectList.module.css

```diff
@@ -18,6 +18,10 @@
   position: relative;
 }
 
+.visuallyHidden {
+  display: none;
```

Comment:
@nathan-ong `display: none` doesn't just visually hide it, it also hides it from screen readers.

Instead I recommend you use `<Box display="visuallyHidden" />`